docs/src/reference/configuration.rst

```diff
@@ -141,455 +177,638 @@ title
+-  default: ``true``
 
-metadata
-~~~~~~~~
 
--  type: string
--  description: Path to a local or remote (S3, HTTP(S), GS) tab-delimited metadata file supported by Augur. Metadata can be uncompressed or compressed.
--  examples:
+priorities:
+-----------
 
-   -  ``data/example_metadata.tsv``
-   -  ``data/example_metadata.tsv.xz``
-   -  ``s3://your-bucket/metadata.tsv.gz``
-   -  ``https://data.nextstrain.org/files/ncov/open/metadata.tsv.gz``
+-  type: object
+-  description: Configures how proximities are calculated, which is used by subsampling schemes which specify it.
+- Valid attributes:
 
-sequences
-~~~~~~~~~
+.. contents::
+   :local:
 
--  type: string
--  description: Path to a local or remote (S3, HTTP(S), GS) FASTA file with \**_un_aligned*\* genome sequences. Sequences can be uncompressed or compressed.
+crowding_penalty
+~~~~~~~~~~~~~~~~
+
+-  type: float
+-  description: used when calculating ``priority scores`` during subsampling to decrease the number of identical samples that are included in the tree during random subsampling to provide a broader picture of the viral diversity in your dataset.
 -  examples:
 
-   -  ``data/example_sequences.fasta``
-   -  ``data/example_sequences.fasta.xz``
-   -  ``s3://your-bucket/sequences.fasta.gz``
-   -  ``https://data.nextstrain.org/files/ncov/open/sequences.fasta.xz``
+.. code:: yaml
 
-aligned
-~~~~~~~
+   priorities:
+     crowding_penalty: 0.0
+     # You may wish to set `crowding_penalty = 0.0` (default value = `0.1`) if you are interested in seeing as many samples as possible that are closely related to your `focal` set. 
 
--  type: string
--  description: Path to a local or remote (S3, HTTP(S), GS) FASTA file with **aligned** genome sequences. Sequences can be uncompressed or compressed.
--  examples:
+.. _title-1:
 
-   -  ``data/aligned.fasta``
-   -  ``data/aligned.fasta.xz``
-   -  ``s3://your-bucket/aligned.fasta.gz``
-   -  ``https://data.nextstrain.org/files/ncov/open/aligned.fasta.xz``
+run_pangolin
+------------
 
-localrules
-----------
+-  type: boolean
+-  description: Enable annotation of Pangolin lineages for a given build’s subsampled sequences.
+-  default: ``false``
 
--  type: string
--  description: Path to a Snakemake file to include in the workflow. This parameter is redundant with ``custom_rules`` and may be deprecated soon.
 
-logistic_growth
----------------
+mask
+----
 
 -  type: object
--  description: Parameters for estimation of logistic clade growth based on logit-transformed clade frequencies.
+-  description: Parameters for masking of invalid or problematic nucleotides in aligned sequences. In addition to the configurable parameters below, the workflow also always masks terminal gaps in the given alignment.
 -  required:
 
-   -  ``delta_pivots``
-   -  ``min_tips``
-   -  ``min_frequency``
-   -  ``max_frequency``
+   -  ``mask_from_beginning``
+   -  ``mask_from_end``
+   -  ``mask_sites``
+- Valid attributes:
 
 .. contents::
    :local:
```

Comment:
_Observable prior to PR, thus non-blocking and can be addressed separately_

This renders a bit redundant:

<img width="120" alt="logistic_growth parameters" src="https://user-images.githubusercontent.com/13424970/162802067-e18c7f94-20f9-406c-b227-a9d5dde334e6.png">

I suggest dropping the "required" bullet point and putting required-ness inline with the attributes, such as like so for `refine`:

<img width="200" alt="refine parameters proposal" src="https://user-images.githubusercontent.com/13424970/162802958-d1e87390-8bef-451c-b885-96d3d9a6e64e.png">

Reply:
Agreed! Fixed in [9b267de](https://github.com/nextstrain/ncov/pull/907/commits/9b267de3ddbe58a2f79b3f0c69af08ebc2a58841) but I didn't add the "[optional]" prefix because in practice none of those attributes are required:

> On the surface we lose the distinction about what parameters were
required, however this was misleading as in practice all required
parameters are defined in our defaults, and Snakemake merges dicts
so that it is perfectly valid to only specify a subset of "required"
parameters.